File: app/src/main/java/app/wallpaper/modules/BaseActivity.java
package app.wallpaper.modules;

import android.os.Bundle;
import android.os.PersistableBundle;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import butterknife.Unbinder;

public abstract class BaseActivity extends AppCompatActivity {

    protected Unbinder unbinder;

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState,
                         @Nullable PersistableBundle persistentState) {
        super.onCreate(savedInstanceState, persistentState);
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
        if (unbinder != null)
            unbinder.unbind();
    }
}

File: app/src/main/java/app/wallpaper/modules/BaseActivity.kt
package app.wallpaper.modules

import androidx.appcompat.app.AppCompatActivity
import butterknife.Unbinder

abstract class BaseActivity : AppCompatActivity() {

    protected var unbinder: Unbinder? = null

    override fun onDestroy() {
        super.onDestroy()
        if (unbinder != null)
            unbinder!!.unbind()
    }
}
